get-latest-tgf.ps1

```diff
@@ -0,0 +1,41 @@
+$ErrorActionPreference = "Stop" #Make all errors terminating
+$env:PATH =$env:PATH+";."
+
+try {
+    $TGF_PATH = (Get-Command tgf.exe).Path
+    Write-Host tgf path: $TGF_PATH
+    $LOCAL_VERSION = (tgf.exe --current-version) | Out-String
+    $LOCAL_VERSION = [regex]::Matches($LOCAL_VERSION,'[0-9]+.[0-9]+.[0-9]+').Value
```

Comment:
You can use the following regular expression: `\d+\.\d+\.\d+`. `.` means "any character", so it needs to be escaped to mean "The `.` charachter". `\d` is equivalent to `[0-9]` (d is for digit).